__tests__/app.test.js

```diff
@@ -73,6 +73,13 @@ describe('backend-express-template routes', () => {
       ...newJersey,
     });
   });
+  it('#PUT /jerseys/:id should update an existing jersey', async () => {
+    const resp = await request(app).put('/jerseys/1').send({
+      cost: '150',
```

Comment:
if the cost field was INT rather than BIGINT, the cost value in this object could be passed in as a number, rather than as a string.